mods/core/src/test/usermanager.unit.test.ts

```diff
@@ -40,35 +40,61 @@ describe('@fonos/core/usermanager/roleOperations', () => {
 })
 
 
-describe('@fonos/core/usermanager/userOperations', ()=>{
-  let findStub;
-  let sampleUser : any[];
-
-  beforeEach(()=>{
-    sampleUser = [{
-      firstName: 'GUEST',
-      lastName: 'GUEST',
-      email: 'GUEST@GMAIL.COM',
-      accessKeyId: '6033ed5bf911e40700000002',
-      role: 'USER',
-      createTime: "Tue Feb 23 2021 09:45:08 GMT-0400 (Atlantic Standard Time)",
-      updateTime: "Tue Feb 23 2021 09:45:08 GMT-0400 (Atlantic Standard Time)",
-      status: 'ACTIVE'
-    }]
-    findStub = sandbox.stub(Mongoose.Model, 'find' ).resolves(sampleUser);
+
+describe('@fonos/core/usermanager/userOperations', () => {
+
+   let sampleUser = new User();
```

Comment:
Use `const` instead of `let`. As far as I can see you are reusing this variable.